java/idl2jni/codegen/im_java.cpp

```diff
@@ -358,8 +358,64 @@ ACE_CDR::LongLong ju_ll(ACE_CDR::ULongLong u)
          ? -static_cast<ACE_CDR::LongLong>(~u+1) : u;
 }
 
-ostream &operator<< (ostream &o, AST_Expression::AST_ExprValue *ev)
+string unicode_escape(int in, bool char_literal = false)
 {
+  switch (in) {
+  case '\'':
+  case '\"':
+  case '\\':
+    return string("\\") + static_cast<char>(in);
+  case '\b':
+    return "\\b";
+  case '\f':
+    return "\\f";
+  case '\n':
+    return "\\n";
+  case '\r':
+    return "\\r";
+  case '\t':
+    return "\\t";
+  }
+
+  if (in <= UCHAR_MAX && isprint(in)) {
+    return string(1, static_cast<char>(in));
+  }
+
+  if (in > 0x10ffff) {
+    return "error: out-of-range character";
+  }
+
+  if (in <= 0xffff) {
+    string escaped = "\\u____";
+    ACE_OS::snprintf(&escaped[2], 5, "%04x", in);
```

Comment:
Missing an include here and below?

Reply:
Yes, somehow I overlooked those GitHub Actions error annotations